lib/rules/disallow-multiple-line-breaks.js

```diff
@@ -46,9 +46,13 @@ module.exports.prototype = {
 
     check: function(file, errors) {
         // Iterate over all tokens (including comments)
-        file.getTokens().forEach(function(token, index, tokens) {
+        file.getTokens().forEach(function(token) {
+            if (token.type === 'Whitespace') {
+                return;
+            }
+
             // If there are no trailing tokens, exit early
-            var nextToken = tokens[index + 1];
+            var nextToken = file.getNextToken(token, {includeComments: true});
```

Comment:
Yes, sure. Now token list contains both `Code`, `Comment` and `Whitespace` tokens.